src/app/code/community/MageHackDay/TwoFactorAuth/etc/config.xml

```diff
@@ -133,6 +133,7 @@
                 <frontend_active>0</frontend_active>
                 <twofactorauth_protected_resources>admin/sales/order,admin/system,all</twofactorauth_protected_resources>
                 <remember_me_duration>2592000</remember_me_duration>
+                <allow_ips></allow_ips>
```

Comment:
Perhaps default to 127.0.0.1

Reply:
::1 and 127.0.0.1 or better the name "localhost"